PKGBUILD

```diff
@@ -0,0 +1,27 @@
+# Maintainer: Jasur Sadikov <jasur@sadikoff.com>
+pkgname=mud
+pkgver=v1.0.2
+pkgrel=1
+pkgdesc="Multi repository git utility. Manage multiple git-repositories simultaneously."
+arch=('any')
+url="https://github.com/jasursadikov/mud"
+license=('MIT')
+depends=('python' 'python-prettytable')
+makedepends=('python-build' 'python-installer' 'git')
+source=("${_pkgname}::git+https://github.com/jasursadikov/mud.git")
+md5sums=('SKIP')
+
+pkgver() {
+    cd "$srcdir/$_pkgname"
+    echo $(grep '^version =' pyproject.toml | cut -d '"' -f2)
```

Comment:
just noticed it says 1.0.1 in your pyproject.toml, but pkgver is already 1.0.2

Reply:
That version is not pushed to pipy, this is the reason :)
But the packaging is not working propertly, I am failing to build :(